apps/web/src/app/(dashboard)/(management)/locatie/[location]/cohorten/[cohort]/[student-allocation]/page.tsx

```
@@ -180,23 +185,14 @@ async function ManageStudentActions({
   }
 
   return (
-    <Dropdown>
-      <DropdownButton outline className="-my-1.5">
-        <EllipsisHorizontalIcon />
-      </DropdownButton>
-      <DropdownMenu anchor="bottom end">
-        <StartExtraProgramForStudentAllocation
-          cohortId={cohortId}
-          personId={personId}
-          locationId={locationId}
-        />
-        <ReleaseStudentAllocation
-          cohortId={cohortId}
-          studentAllocationId={studentAllocationId}
-          locationId={locationId}
-        />
-      </DropdownMenu>
-    </Dropdown>
+    <ManageStudentActionsDropdown
+      cohortId={cohortId}
+      studentAllocationId={studentAllocationId}
+      locationId={locationId}
+      personId={personId}
+      cohorts={filteredCohorts}
+      canMoveStudentAllocation={allocation ? !allocation.certificate : false}
```

This allocation couldn't be null, right? Otherwise, this page could not exist. So maybe throw an error if an allocation is not found, and this can simply become `!allocation.certificate`.